.github/workflows/service.yml

```diff
@@ -0,0 +1,31 @@
+name: Java Library Services + Models
+
+on: [ workflow_dispatch ]
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    name: Generate Models + Services
+    steps:
+      - uses: actions/checkout@v3
+      - run: make services
+      - name: Set PR variables
+        id: vars
+        run: |
+          cd target/spec
+          echo ::set-output name=pr_title::"Update models + services"
+          echo ::set-output name=pr_body::"OpenAPI spec or templates produced new models + services on $(date +%d-%m-%Y) \
+            by [commit](https://github.com/Adyen/adyen-openapi/commit/$(git rev-parse HEAD))."
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
+          committer: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
+          author: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
+          base: develop
+          branch: automation/services
+          title: ${{ steps.vars.outputs.pr_title }}
+          body: ${{ steps.vars.outputs.pr_body }}
+          add-paths: |
+            src/main/java/com/adyen/model
```

Comment:
just to understand, if this script is for both /service and /model, do we plan to remove the `models.yml` file?